bolt/spark/shapes.py

```diff
@@ -30,6 +38,14 @@ def shape(self):
         return self._barray.shape[:self._barray.split]
 
     def reshape(self, *shape):
+        """
+        Reshape just the keys of a BoltArraySpark, returning a 
+        new BoltArraySpark.
+
+        Parameters                                                                           
+        ----------                                                                           
+        shape : a tuple of new proposed axes.
```

Comment:
Yup! Sorry for the inconsistency. Will def shore that up
On Mon, Jul 6, 2015 at 23:33 Jeremy Freeman <notifications@github.com>
wrote:

> In bolt/spark/shapes.py
> <https://github.com/bolt-project/bolt/pull/54#discussion_r34005643>:
>
> > @@ -30,6 +38,14 @@ def shape(self):
> >          return self._barray.shape[:self._barray.split]
> >
> >      def reshape(self, *shape):
> > +        """
> > +        Reshape just the keys of a BoltArraySpark, returning a
> > +        new BoltArraySpark.
> > +
> > +        Parameters
> > +        ----------
> > +        shape : a tuple of new proposed axes.
>
> Here and elsewhere, the format should be
>
> name : type
>    description...
>
> so here it should be
>
> shape : tuple
>     The set of new proposed axes.
>
> make sense?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bolt-project/bolt/pull/54/files#r34005643>.
>